docs/files/theory.rst

```diff
@@ -0,0 +1,74 @@
+.. _basictheory:
+
+Basic theory
+============
+
+The MANDYOC code simulates thermochemical convection of the Earth's mantle. The following sections explain which equations are being solved by the code and the numerical approach that was used.
+
+Basic equations
+---------------
+
+To simulate mantle thermo-mechanical convection, we adopted the formulation for non-Newtonian fluids together with the Boussinesq approximation to solve the following equations of conservation of mass (:eq:`mass-conservation`), momentum (:eq:`momentum-conservation`) and energy (:eq:`energy-conservation`) :cite:`zhong2007`.
+
+.. math:: 
+	:label: mass-conservation
+
+	u_{i,i}=0 
+
+.. math:: 
+	:label: momentum-conservation
+
+	\sigma_{ij,j}+g\rho_0(1-\alpha(T-T_0))=0
```

Comment:
Please, see the equation for \rho in the JOSS paper.

Reply:
I found the expression \rho=\rho_{0}(1-\alpha(T-T_{0})), the same as on the line 21, but g was missing and "i" subscript.